src/Migration/AbstractMigrationBuilder.php

```diff
@@ -0,0 +1,339 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Db\Migration;
+
+use Yiisoft\Db\Schema\ColumnSchemaBuilder;
+use Yiisoft\Db\Schema\Schema;
+use Yiisoft\Db\Schema\SchemaInterface;
+
+/**
+ * AbstractMigrationBuilder contains shortcut methods to create instances of {@see ColumnSchemaBuilder}.
+ *
+ * These can be used in database migrations to define database schema types using a PHP interface. This is useful to
+ * define a schema in a DBMS independent way so that the application may run on different DBMS the same way.
+ *
+ * For example, you may use the following code inside your migration files:
+ *
+ * ```php
+ * $this->createTable(
+ *     'example_table',
+ *     [
+ *         'id' => $this->primaryKey(),
+ *         'name' => $this->string(64)->notNull(),
+ *         'type' => $this->integer()->notNull()->defaultValue(10),
+ *         'description' => $this->text(),
+ *         'rule_name' => $this->string(64),
+ *         'data' => $this->text(),
+ *         'created_at' => $this->datetime()->notNull(),
+ *         'updated_at' => $this->datetime(),
+ *     ],
+ * );
+ * ```
+ */
+abstract class AbstractMigrationBuilder
+{
+    public function __construct(private SchemaInterface $schema)
+    {
+    }
```

Comment:
I think it's easier to read if we'll format it like the following:

```suggestion
    public function __construct(
        private SchemaInterface $schema
    )
    {
    }
```

Reply:
I think: not for short lines